101-recoveryservices-vault-create/README.md

```diff
@@ -0,0 +1,14 @@
+# Create Recovery Services Vault
```

Comment:
@nilaydshah Let's call this folder 101-recovery-services-vault-create for readability